src/Compilers/CSharp/Portable/CSharpResources.resx

```diff
@@ -6067,4 +6082,7 @@ To remove the warning, you can use /reference instead (set the Embed Interop Typ
   <data name="WRN_GeneratorFailedDuringInitialization_Title" xml:space="preserve">
     <value>Generator failed to initialize.</value>
   </data>
+  <data name="ERR_AssignmentInitOnly" xml:space="preserve">
+    <value>Init-only member '{0}' can only be assigned from a constructor, object initialization or 'with' expression of that type.</value>
```

Comment:
>Init-only member [](start = 11, length = 16)

Is there a concept of "Init-only **member**"?

Reply:
The spec was written in those terms. We now updated it, I'll update here too.

---
In reply to: [415097040](https://github.com/dotnet/roslyn/pull/43275#discussion_r415097040) [](ancestors = 415097040)